src/index.js

```diff
@@ -201,6 +202,11 @@ function init(options, outputDirsMap, externalGulp) {
     computedOptions[taskName] = defaults({}, options[taskName], taskOptions);
   });
 
+  if (computedOptions.js.useES6) {
+    computedOptions.js.browserifyTransforms.push([babelify, { presets: [require('babel-preset-es2015')] }]);
```

Comment:
I would prefer to set this options more straight forward as they are not so complicated, just put it in configuration object above. It will give user better idea of what is going on, and there will be no case that configuration of js task affects eslint configuration.